packages/backend/package.json

```diff
@@ -9,14 +9,18 @@
 	"license": "MIT",
 	"scripts": {
 		"build": "tsc -p tsconfig.build.json",
-		"webpack": "webpack --env mode=development && cp ./lib/bundle.cjs ../backend-bundle/bundle.cjs",
+		"webpack": "run-script-os",
+		"webpack:default": "webpack --env mode=development && cp ./lib/bundle.cjs ../backend-bundle/bundle.cjs",
 		"webpack:prod": "webpack --env mode=production && cp ./lib/bundle.cjs ../backend-bundle/bundle.cjs",
-		"applyPatches": "patch -f -p0 < ./electron-fetch.patch || true && patch -f -p0 --forward --binary < ./parse-duration.patch || true && patch -f -p0 --forward --binary < ./parse-duration-esm.patch || true",
+		"webpack:windows": "webpack --env mode=development && xcopy .\\lib\\bundle.cjs ..\\backend-bundle\\ /Y",
+		"webpack:prod:windows": "webpack --env mode=production && xcopy .\\lib\\bundle.cjs ..\\backend-bundle\\ /Y",
+		"applyPatches": "run-script-os",
+		"applyPatches:default": "patch -f -p0 < ./electron-fetch.patch || true && patch -f -p0 --forward --binary < ./parse-duration.patch || true && patch -f -p0 --forward --binary < ./parse-duration-esm.patch || true",
+		"applyPatches:windows": "git apply ./electron-fetch-git.patch --whitespace=fix --reject --verbose --no-index --ignore-space-change --inaccurate-eof || cd .",
```

Comment:
Should we just use git apply on all systems to keep things simple?

Reply:
I totally agree that we should use one tool instead of two for the same purpose but **I think** patch files differ in format between those generated by patch and git.
I'd say: let's do this but not in scope of this MR but separately.

Reply:
What's more - I still have a problem with handling git's patch correctly (on Windows at least). Applying works fine on windows machine that patch was created on but fails on CI or another windows machine. I suppose git may be changing encoding? I am not sure yet.

Reply:
Ok sounds good. Does the issue on CI break our Windows CI in any way?

Reply:
It should not. E2E tests pass, other checks too.